devel/libevent/Portfile

```diff
@@ -20,12 +21,14 @@ long_description    The libevent API provides a mechanism to execute \
 
 homepage            http://libevent.org
 
-checksums           rmd160  f23ad93d555089535085c99f3f8cbb9ebae4fd53 \
-                    sha256  d53e88f2b31fbccab93f6b1e54fe7f4c1d536b6d8250c95cd42dbc14f958baeb
+conflicts           libev
+
+checksums           rmd160 a8a158035d7677e9db00f63fab45fbadd3da41df \
+                    sha256 e864af41a336bb11dab1a23f32993afe963c1f69618bd9292b89ecf6904845b0
```

Comment:
We are now adding the `size` parameter to `checksums`.

Reply:
Just for my curiosity, when did we start using `size`?

Reply:
It's been about a year or so.

Reply:
I think almost two, FWIW.